pkg/api/validation/validation.go

```diff
@@ -2034,7 +2035,7 @@ func ValidateAppArmorPodAnnotations(annotations map[string]string, spec *api.Pod
 		if !strings.HasPrefix(k, apparmor.ContainerAnnotationKeyPrefix) {
 			continue
 		}
-		if !utilflag.DefaultFeatureGate.AppArmor() {
+		if !utilfeature.DefaultFeatureGate.Enabled(features.AppArmor) {
```

Comment:
This is really ugly and generally wrong.  Not new in this pull but the link here should be snipped.  This is an admission choice, not a general validation choice.